keybinding.go

```diff
@@ -15,6 +27,68 @@ type keybinding struct {
 	handler  func(*Gui, *View) error
 }
 
+var (
+	ErrorNotExist = errors.New("no such keybind")
+)
+
+func Parse(input string) (interface{}, Modifier, error) {
```

Comment:
And add a comment to `Parse`, `ParseAll`, 'MustParse' and `MustParseAll`

Reply:
Will do!